docroot/themes/humsci/humsci_basic/src/scss/components/_card.scss

```diff
@@ -147,14 +147,14 @@
 
     &--date {
       // only apply calendar icon when the label exists
-      .views-label-field-hs-event-date + .field-content {
+      [class*="views-label-field-hs-event-date"] + .field-content {
```

Comment:
We are using classes generated by Drupal for the styling of the icons. Targeting `.views-label-field-hs-event-date` is fine for the highlight events but the icons were not displaying for the list of upcoming seminars. I discovered upcoming seminars has a different class name of `.views-label-field-hs-event-date-5`. I'm unsure how Drupal generates the `-5` and assume this number could be different when the date stacked horizontal pattern is used on another view. The change introduced was the only way I could think of targeting all potential scenarios.